tutorials/pdes/heat-1-an-introduction.md

```diff
@@ -0,0 +1,179 @@
+# The Heat Equation
+
+**Authors**: Manuel Madeira, David Carvalho
+
+**Reviewers**: Fábio Cruz
+
+
+In this tutorial, we delve into the problem of ... 🔥 **heat diffusion** on a plate! 🔥
+
+We will take you step by step up to a point where you can gauge the potential of Neural Networks
+-- in both strength and versatility -- in solving Partial Differential Equations:
+
+1. **The Heat Equation**: introduces the physics behind heat diffusion.
+
+2. **Finite-Differences**: classical numerical routine to approximate its solution across a 2D plate.
+
+3. **Physics-Inspired Neural Network**: with that estimate of the solution as a *benchmark*, see how **Machine-Learning algorithms** fare for the same situation, by deploying a PINN
+(Physics-Inspired Neural Network).
+
+4. **Generalized Neuro-Solver***: explore further into how versatile these Neural Networks are in
+handling **more complex geometries** than the 2D domain and **varying
+initial/boundary conditions**;
+
+This field is fast-growing but, most importantly is at its *blissful infancy* stage.
+Our dream, rooted in a belief, is that Machine Learning may be refined to excel
+in extremely complex and customizable cases.
+
+This sophistication could potentially fare better in performance than
+every other classical solver, allowing us to go to lands classical frameworks have never
+granted us so far!
+
+Alright. 🔥 *Let's get hot in here!* 🔥
```

Comment:
Yes: I think you can summarize this text a very shrort intro on the cover page.

Reply:
I also think we need to have a line like with something like this: 

"The world of classical simulation and the world AI are merging into a broader form of computational solutions for science and engineering. In a previous tutorial, we showed how you could use Inductiva's simulation API to generate synthetic data for training a GNN.In this tutorial, we will go in a slightly different direction and we guide you through an example one of the most exciting technical outcomes of this marriage between classical simulation and AI: Physically Informed Neural Networks, or PINNs for short. The most notable difference between using PINNs vs using more generic neural networks is that, in PINNs, we explicitly encode knowledge about the the physical laws in the training loss of the neural network, allowing it to more quickly converge to functions that are physically sound. 

For this tutorial, we will use a simple .... bla bla heat
The structure of this tutorial is..."

Reply:
Thanks!